station.localization.json

```diff
@@ -4502,6 +4502,33 @@
       "sv": "",
       "fr": ""
     },
+    {
+      "ident_ios": "OffsetCorrection.Dialog.Calibration.EnterTemperature",
```

Comment:
Please use existing strings for android calibration_enter_temperature, calibration_enter_pressure and calibration_enter_humidity. 

This pattern {%@^%1$s} works like this {foriOS^forAndroid}. So it will work the same

Reply:
removed, please check again